rviz_visual_testing_framework/src/visual_test_fixture.cpp

```diff
@@ -51,13 +52,17 @@ void VisualTestFixture::SetUpTestCase()
   visualizer_app_->setApp(qapp_);
 
   visualizer_app_->init(argc, argv);
+
+  std::string package_share_directory = ament_index_cpp::get_package_share_directory(
+    "rviz_visual_testing_framework");
   if (VisualTest::generateReferenceImages()) {
     visualizer_app_->loadConfig(QDir::toNativeSeparators(
-        QString::fromStdString(std::string(_SRC_DIR_PATH) + "/visual_tests_default_config.rviz")));
+        QString::fromStdString(package_share_directory +
+        "/rviz/visual_tests_default_config.rviz")));
```

Comment:
You should double-check, but this doesn't look right. I think the config files are currently being installed to the subdirectory "rviz/config/" not just "rviz". I suggest remove the "rviz" suffix from the CMakeLists.txt install rule and updating this line to
```suggestion
        "/config/visual_tests_default_config.rviz")));
```


Same for the other config file path below.